include/networkit/scd/LocalT.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef NETWORKIT_SCD_LOCAL_T_HPP_
+#define NETWORKIT_SCD_LOCAL_T_HPP_
+
+#include <networkit/scd/SelectiveCommunityDetector.hpp>
+
+namespace NetworKit {
+
+/**
+ * The local community expansion algorithm optimizing the T measure.
+ *
+ * This implements the algorithm published in:
+ *
+ * Fagnan, J., Zaiane, O., & Barbosa, D. (2014).
+ * Using triads to identify local community structure in social networks.
+ * In 2014 IEEE/ACM International Conference on Advances in Social Networks Analysis and Mining
+ * (ASONAM) (pp. 108–112). https://doi.org/10.1109/ASONAM.2014.6921568
+ */
+class LocalT : public NetworKit::SelectiveCommunityDetector {
```

Comment:
As above.

Reply:
Fixed in 56c52c5639441be8bda89fbac450c6c1689c709b